xdocs/usermanual/functions.xml

```diff
@@ -1499,6 +1502,36 @@ becomes:
 </properties>
 </component>
 
+
+<component index="&sect-num;.5.22" name="__timeShifting">
+<description>
+    <p>The timeShifting function returns a date in various formats with the specified amount of seconds/minutes/hours/days/months added</p>
+    </description>
+<properties>
+        <property name="Format" required="No">
+        The format to be passed to DateTimeFormatter.
+        If omitted, the function use milliseconds since epoch format.
+        </property>
+        <property name="Date to shift" required="No">
+        Indicate the date with the format from first paramater to shift
+        If omitted, the date is set to now
+        </property>
+        <property name="value to shift" required="No">
+        Indicate the specified amount of seconds/minutes/hours/days/mounths to add according to this usage :
+        <ul>
+        <li><code>5s</code> for Second value</li>
+        <li><code>5m</code> for Minute value</li>
+        <li><code>5H</code> for Hour value</li>
+        <li><code>5d</code> for Day value</li>
+        <li><code>5M</code> for Month value</li>
+        </ul>
+        To shift in the pass, you just have to indicate <code>-5d</code>  to remove 5 days.
```

Comment:
Would it be useful, to specify more complex amount of times like `5d7H` or `5 days 7 minutes`?